node/src/accountant/payment_adjuster/mod.rs

```diff
@@ -302,60 +309,48 @@ impl PaymentAdjusterReal {
         merged
     }
 
-    fn resolve_current_iteration_result(
-        &mut self,
-        adjustment_iteration_result: AdjustmentIterationResult,
-    ) -> RecursionResults {
-        let remaining_undecided_accounts = adjustment_iteration_result.remaining_undecided_accounts;
-        let here_decided_accounts = match adjustment_iteration_result.decided_accounts {
-            LowGainingAccountEliminated => {
-                if remaining_undecided_accounts.is_empty() {
-                    return RecursionResults::new(vec![], vec![]);
-                }
-
-                vec![]
-            }
-            SomeAccountsProcessed(decided_accounts) => {
-                if remaining_undecided_accounts.is_empty() {
-                    return RecursionResults::new(decided_accounts, vec![]);
-                }
-
-                self.adjust_remaining_unallocated_cw_balance_down(&decided_accounts);
-                decided_accounts
-            }
-        };
-
-        let down_stream_decided_accounts = self.propose_adjustments_recursively(
-            remaining_undecided_accounts,
-            ServiceFeeOnlyAdjustmentRunner {},
-        );
+    fn is_cw_balance_enough_to_remaining_accounts(
```

Comment:
A minor change here: instead of `to_remaining_accounts` you want `for_remaining_accounts`.